public/model/epi-data-store.js

```diff
@@ -17,105 +17,148 @@ var utc_date = function(year, month, day) {
-      });
+      .catch(function(err) { alert('Error getting case data! ' + err); });
   },
 
-  // Return a recent epi data record. If there are multiple records that include
-  // the given `date`, returns the one that has data on the most regions. If
-  // there are no records that include `date`, returns the most recent record
-  // that occurred before the date.
-  get_best_recent_epi_data: function(date) {
+  // Returns recent epi data records for the given countries. If there are multiple records that
+  // include the given `date`, returns the one that has data on the most admins. If there are no
+  // records that include `date`, returns the most recent record that occurred before the date.
+  //
+  // TODO(jetpack): The returned records are not guaranteed to be for the same time interval. For
+  // example, if the most recent data for Brazil is from 2016-01-01 and for Colombia is from
+  // 2016-03-15, both data sets will be returned. This is probably misleading, but leaving for now
+  // as we aren't sure what the UX for epi data should be yet.
+  //
+  // TODO(jetpack): binary search instead of linear scan.
+  get_best_recent_epi_data: function(country_codes, date) {
     if (!(date instanceof Date)) {
       console.error('Expected a Date, but got:', date);
       return null;
     }
-    // TODO(jetpack): binary search instead of linear scan.
-    var matching_records = _.filter(this.data_by_date_interval_and_region, function(record) {
-      return record.start_time <= date && date < record.end_time;
-    });
-    if (!_.isEmpty(matching_records)) {
-      return _.maxBy(matching_records, function(r) { return _.size(r.region_case_data); });
-    }
-    // No epi records contain `date`. Find the record with the most recent
-    // `start_time` that occurred before the date.
-    // TODO(jetpack): binary search instead of linear scan.
-    console.log('EpiDataStore doesnt have matching data for', date,
-                ', just looking for most recent data now..');
-    return _.findLast(this.data_by_date_interval_and_region, function(record) {
-      return record.start_time < date;
-    });
+    var result = [];
+    _.forEach(country_codes, (function(country_code) {
+      var matching_records = _.filter(this.records_by_country[country_code], function(record) {
+        return record.start_time <= date && date < record.end_time;
+      });
+      if (!_.isEmpty(matching_records)) {
+        result.push(_.maxBy(matching_records, function(r) { return _.size(r.admin_case_data); }));
+      } else {
+        // No epi records contain `date`. Find the record with the most recent `start_time` that
+        // occurred before the date.
+        console.log('EpiDataStore doesnt have matching data for', date,
+                    ', just looking for most recent data now..');
+        var recent = _.findLast(this.records_by_country[country_code], function(record) {
+          return record.start_time < date;
```

Comment:
Yeah, had the same thought. I think what to do display when we don't have recent data is a tricky product/UX question. Even more complicated when considering multiple data sources (e.g. weather data from last week, case data from a month ago, mobility data from 3 months ago).

Anyway. Added a TODO for now :).